app/components/contributors/edit_component.html.erb

```diff
@@ -1,20 +1,18 @@
 <%= tag.div data: contributors_data do %>
   <%= render Elements::Forms::ToggleComponent.new(form:, field_name: :role_type, label_classes: 'fw-bold',
                                                   label: t('works.edit.fields.contributors.role_type.label'), container_classes: 'mb-4') do |component| %>
-    <% component.with_toggle_option(form:,
+    <% component.with_left_toggle_option(form:,
```

Comment:
There's always going to be 2 toggle options, so renaming to `left_toggle_option` and `right_toggle_option`.

Reply:
What about RTL langs? 😆

Reply:
@jcoyne Coyne.

Reply:
They are in the current designs.